build.psm1

```diff
@@ -158,7 +158,30 @@ function Start-PSBuild {
     }
 
     # save Git description to file for PowerShell to include in PSVersionTable
-    git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60 > "$psscriptroot/powershell.version"
+    $powershellVersion = git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60
+    $powershellVersion > "$PSScriptRoot/powershell.version"
```

Comment:
I see. It might be fine if you generated the file alongside other test artifacts, but I would not create it in `$PSHome`.